shells/backends/gcp_tf_backend/test_requirements.txt

```diff
@@ -0,0 +1,7 @@
+nose
```

Comment:
Please set versions for packages

Reply:
Hello @qualidan , could you help me with this, I checked AWS and Azure they also haven't versions.